hare/hare.go

```diff
@@ -254,12 +269,13 @@ func (h *Hare) onTick(ctx context.Context, id types.LayerID) {
 	instID := instanceID(id)
 	c, err := h.broker.Register(ctx, instID)
 	if err != nil {
-		logger.With().Warning("could not register consensus process on broker", log.Err(err))
+		logger.With().Error("could not register consensus process on broker", log.Err(err))
+		err = errors.New("could not register consensus process on broker")
```

Comment:
What's the point of changing the error here? We do nothing with it other than check for nilness.